### DISCUSSION TITLE
What is the arch difference between bolt and original chronos?

### DISCUSSION THREAD

**Discussion by 2catycm** [amazon-science/chronos-forecasting#240] [Q&A] [answered]
In the readme and doc, I got very little information of the details of bolt, except that it is better, I know nothing.

**Comment by abdulfatir** [comment, marked as answer]
Hey @2catycm, the best resource for details on the Chronos-Bolt model's architecture is the code. See this: https://github.com/amazon-science/chronos-forecasting/blob/main/src/chronos/chronos_bolt.py

In short, Chronos-Bolt retains the T5 architecture but does patch-level modeling instead. To enable this we need:
- an input patch embedding network which converts patches to `d_model`-dimensional representations.
- an output patch embedding network which maps the model's representations to multi-step quantiles. 

If you have specific questions, please feel free to ask in this thread.

**Reply by Mhdaw** [reply]
@abdulfatir 
hey, so the bolt series do not perform the tokenization? and perform a patching?

**Reply by jayjunjieqiu** [reply]
Exactly
